.github/workflows/countline.py

```diff
@@ -0,0 +1,22 @@
+
+import pytest
+import pandas as pd
+from io import StringIO
+import os
+
+test_output_path = os.path.dirname(os.path.abspath(__file__)) + \
```

Comment:
Use `os.path.split` and `os.path.join` to create the path. If this is run on a windows system it will fail. Also use `os.path.expanduser` to limit the unlikely case where someone uses a ~ in the path name.

Reply:
Consider using [argparse](https://docs.python.org/3/library/argparse.html) for specifying the directory with an optional default.

Reply:
Done